src/main/java/net/wasdev/wlp/ant/InstallFeatureTask.java

```diff
@@ -67,7 +200,23 @@ private void doInstall() throws Exception {
         if (whenFileExists != null) {
             command.add("--when-file-exists=" + whenFileExists);
         }
-        command.add(name);
+        if (directoryPath != null) {
+            command.add("--location=" + directoryPath);
+        }
+        if (downloadOnly != null) {
+            command.add("--downloadOnly=" + downloadOnly);
```

Comment:
Yes, we're still using featureManager, but, doesn't it download all the dependencies automatically with the value --downloadOnly=all?